<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSend</class>
 <widget class="QWidget" name="PageSend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FCD535;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #474D57;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235,235,235);
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../../res.qrc">
      <normaloff>:/res/ic/res/ic/arowLeft.png</normaloff>:/res/ic/res/ic/arowLeft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="tokenComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid lightgray;
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
    min-width: 6em; 
}
QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 5px; /* same radius as the QComboBox */
    border-bottom-right-radius: 5px;
}
QComboBox::drop-down::hover {
	background-color: rgba(0, 0, 0, 16);
}
QComboBox::drop-down::pressed {
    background-color: rgba(0, 0, 0, 64);
}
QComboBox::down-arrow {
    image: url(:/res/ic/res/ic/dropDownArrow.png);
	width: 12px;
    height: 12px;
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QListView::item {
	height: 31px;
	background-color: white;
	color: black;
}
QListView::item:selected:!active, QListView::item:selected:active {
	background-color: lightgrey;
	color: black;
}
QListView::item::hover {
	background-color: grey;
	/*font: url(:/Resources/Fonts/GameJam.ttf);*/
	color: black;
}
</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="recipientAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
QLineEdit{ 
	color:black; 
	border-style: solid;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
    padding-left: 4px;
    padding-right: 4px;	
}
</string>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Recipient address</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
QLineEdit{ 
	color:black; 
	border-style: solid;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
    padding-left: 4px;
    padding-right: 4px;	
}
</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="qrPushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../res.qrc">
     <normaloff>:/res/ic/res/ic/qr.png</normaloff>:/res/ic/res/ic/qr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pastePushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../res.qrc">
     <normaloff>:/res/ic/res/ic/paste.png</normaloff>:/res/ic/res/ic/paste.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sendingLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-radius: 5px;
border-width: 1px;
border-color: lightgrey;
background-color: rgb(255, 254, 173);</string>
   </property>
   <property name="text">
    <string>Copied to clipboard</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
